pyproject.toml

```diff
@@ -100,7 +100,7 @@ ignore_errors = true
 
 [tool.black]
```

Comment:
onnx doesn’t use black anymore. Maybe we should remove the black entries?

Reply:
Can